vmdb/app/models/ext_management_system.rb

```diff
@@ -92,7 +92,13 @@ def self.supported_types_and_descriptions_hash
   include AuthenticationMixin
   include Metric::CiMixin
   include AsyncDeleteMixin
+  include ReservedMixin
 
+  attr_via_reserved :realm
+  attr_via_reserved :security_protocol
```

Comment:
I think we only want to do this for patch releases.  Since this is a real release, these should be real columns with a migration.

@Fryguy, does that sound right?